javascript/javascript1/week3/note-pad.js

```diff
@@ -18,7 +18,7 @@ function getNote(id) {
 		if (notes[i].id === id) {
 			return notes[i];
 		} else {
-			return 'error';
+			return 'error!';
```

Comment:
instead of return error as a string you can throw it
`throw new Error('note not found')`